stories/core/DropdownMenu.stories.tsx

```diff
@@ -60,13 +61,17 @@ export const Basic: Story<DropdownMenuProps> = (args) => {
     <MenuItem key={2} onClick={onClick(2, close)}>
       Item #2
     </MenuItem>,
-    <MenuItem key={3} onClick={onClick(3, close)} disabled>
+    <MenuDivider key={3} />,
```

Comment:
Maybe this should be a new story, as it's not really "Basic"

Reply:
Added.